Assignments/stu5/assignments.py

```diff
@@ -104,3 +106,11 @@ def ex7():
 # Place your functions here...
 #
 
+
+def sort_people(people_list):
```

Comment:
The `sort_people()` function needs to support three arguments: `sort_people(people_list, 'weight', 'desc')`

It needs to be dynamic and sort people by various fields in asc or desc order.


Here are a few use cases that the sort_people() function has to support:

`sort_people(people_list, 'weight', 'desc')` ==> Returns a list sorted in desc order by weight
`sort_people(people_list, 'weight', 'asc')` ==> Returns a list sorted in asc order by weight
`sort_people(people_list, 'age', 'asc')` ==> Returns a list sorted in asc order by age
`sort_people(people_list, 'age', 'desc')` ==> Returns a list sorted in desc order by age
`sort_people(people_list, 'name', 'desc')` ==> Returns a list sorted in desc order by name

Reply:
UPDATED